### PR TITLE
confluence: run `cookstyle -a .`

### PR DESCRIPTION
**Pull request by zmaupin**
## Description

`cookstyle -a .` and manual fixes for cookstyle improvements.

Ran from latest chef/chefdk docker image.
```
root@1b30b6bf5af8:/home# chef --version
ChefDK version: 4.4.27
Chef Infra Client version: 15.3.14
Chef InSpec version: 4.16.0
Test Kitchen version: 2.3.3
Foodcritic version: 16.1.1
Cookstyle version: 5.6.2
```
### Issues Resolved

https://github.com/sous-chefs/confluence/issues/187

### Check List
- [ ] All tests pass. See https://github.com/sous-chefs/apache2/blob/master/TESTING.md
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by zmaupin** [comment]
I'm going to see if I can request that the latest cookstyle version uploaded to Rubygems. That's 5.9.0. Then I'll run this again.

File: test/fixtures/cookbooks/confluence_test/recipes/postgresql.rb [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-  node.normal['postgresql']['dir'] = '/etc/postgresql/9.3/main'
-  node.normal['postgresql']['server']['service_name'] = 'postgresql'
-elsif node['platform'] == 'centos'
-  node.normal['postgresql']['enable_pgdg_yum'] = true
-  node.normal['postgresql']['client']['packages'] = %w(postgresql93 postgresql93-devel)
-  node.normal['postgresql']['server']['packages'] = ['postgresql93-server']
-  node.normal['postgresql']['contrib']['packages'] = ['postgresql93-contrib']
-  node.normal['postgresql']['server']['service_name'] = 'postgresql-9.3'
-  node.normal['postgresql']['setup_script'] = 'postgresql93-setup'
+if platform_family?('rhel')
```
Comment (JohnRoesler) [inline test/fixtures/cookbooks/confluence_test/recipes/postgresql.rb]: i think there is an update to cookstyle that fixes this bug - currently it's putting some of the platform names wrong

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a CHANGELOG entry.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by damacus** [comment]
Thanks!

**Pull request merged**